.pipelines/pipeline.yaml

```diff
@@ -53,555 +53,555 @@ stages:
             displayName: "Set environmental variables"
             condition: always()
   - ${{ if contains(variables['Build.SourceBranch'], 'refs/pull') }}:
```

Comment:
Is the only change a tab difference? Hard to see from the diff

Reply:
yeah tab diff from formatter, reverting